releasenotes/README.md

```diff
@@ -12,10 +12,18 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v50 release notes.](./v50.md)
 
-## February 25, 2020
+## March 4, 2021
 
 These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in the next official release. This list isn't final and is subject to change.
 
+* CHANGE: As of v51 all `source` and `mdapi` use REST as the default api. You can switch between REST and SOAP with the `restDeploy` config value or the `SFDX_REST_DEPLOY` environment variable. Set those to `true` for REST and `false` for SOAP. (GitHub Issues [#860](https://github.com/forcedotcom/cli/issues/860), [#870](https://github.com/forcedotcom/cli/issues/870), [#872](https://github.com/forcedotcom/cli/issues/872), [#884](https://github.com/forcedotcom/cli/issues/884))
```

Comment:
Minor edit: 

CHANGE: As of v51, all `force:source` and `force:mdapi` commands use REST API by default to deploy. Previously they used SOAP API by default. Set the `restDeploy` config value or `SFDX_REST_DEPLOY` environment variable to `false` to switch back to SOAP.  (then github stuff)